webapp/channels/src/components/toast_wrapper/toast_wrapper.tsx

```diff
@@ -281,6 +287,12 @@ export class ToastWrapperClass extends React.PureComponent<Props, State> {
         }
     };
 
+    hideScrollToBottomToast = () => {
+        if (this.props.onScrollToBottomToastDismiss) {
+            this.props.onScrollToBottomToastDismiss();
+        }
+    };
```

Comment:
This can be simplified to something like this.
```suggestion
    hideScrollToBottomToast = () => this.props.onScrollToBottomToastDismiss?.();
```